frontend/src/components/WhoTargetsPanel/IntegratedWhoTargetsPanel.tsx

```
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useWhoTargetsWithWeeklyPlan } from '../../hooks/useWhoTargetsWithWeeklyPlan';
+import type { TargetsRequest } from '../../api/premium/types';
+import { WhoTargetsPanel } from '../WhoTargetsPanel';
+import { WhoTargetsSkeleton } from './Skeleton';
+import { WhoTargetsErrorState } from './ErrorState';
+import { WhoTargetsEmptyState } from './EmptyState';
+
+import type { TargetsRequest } from '../../api/premium/types';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TargetsRequest'. Consider to delete it or rename it.

'TargetsRequest' is defined here:


(lint/suspicious/noRedeclare)